index.bs

```diff
@@ -902,13 +906,29 @@ remote-playback-modify-request message with the following values:
 : controls
 :: Updated controls as defined in {#remote-playback-state-and-controls}
 
+: remoting-audio
```

Comment:
Can this be omitted if we only support video?

Reply:
Remote playback of audio streams is a use case we'd like to see supported. I hope there is implementer interest!